src/context/cartContext.js

```diff
@@ -0,0 +1,44 @@
+import { createContext, useContext, useReducer, useEffect } from "react";
+import { initialCartState, cartReducer } from "../redux";
```

Comment:
We are'nt actually using redux, so the name feels misleading

Reply:
Yes, but the work is similar to redux right? Like I followed this folder structure which was shared by an MA and hence that naming.

![Screenshot 2022-03-10 at 11 37 15](https://user-images.githubusercontent.com/62737267/159117599-0ef574a5-dffd-4165-b15b-3d294bb28f8f.png)

Reply:
Oh okay, i wasnt aware of that, thanks